sktime/forecasting/exp_smoothing.py

```diff
@@ -0,0 +1,179 @@
+        Should the Box-Cox transform be applied to the data first? If 'log'
+        then apply the log. If float then use lambda equal to float.
+    remove_bias : bool, optional
+        Remove bias from forecast values and fitted values by enforcing
+        that the average residual is equal to zero.
+    use_basinhopping : bool, optional
+        Using Basin Hopping optimizer to find optimal values
+
+    References
+    ----------
+    [1] Hyndman, Rob J., and George Athanasopoulos. Forecasting: principles
+        and practice. OTexts, 2014.
+    """
+
+    def __init__(
+        self,
+        trend=None,
+        damped=False,
+        seasonal=None,
+        seasonal_periods=None,
+        smoothing_level=None,
+        smoothing_slope=None,
+        smoothing_seasonal=None,
+        damping_slope=None,
+        optimized=True,
+        use_boxcox=False,
+        remove_bias=False,
+        use_basinhopping=False,
+    ):
+        # Model params
+        self.trend = trend
+        self.damped = damped
+        self.seasonal = seasonal
+        self.seasonal_periods = seasonal_periods
+
+        # Fit params
+        self.smoothing_level = smoothing_level
+        self.optimized = optimized
+        self.smoothing_slope = smoothing_slope
+        self.smooting_seasonal = smoothing_seasonal
+        self.damping_slope = damping_slope
+        self.use_boxcox = use_boxcox
+        self.remove_bias = remove_bias
+        self.use_basinhopping = use_basinhopping
+        super(ExpSmoothingForecaster, self).__init__()
+
+    def fit(self, y, fh=None):
+        """
+        Fit to training data.
+
+        Parameters
+        ----------
+        y : pandas.Series
+            Target time series to which to fit the forecaster.
+        fh : array-like, optional (default=[1])
+            The forecasters horizon with the steps ahead to to predict.
+        Returns
+        -------
+        self : returns an instance of self.
+        """
+
+        y = validate_y(y)
+        self._set_fh(fh)
+
+        # update observation horizon
+        self._set_obs_horizon(y.index)
+
+        # Fit forecaster.
+        self._estimator = ExponentialSmoothing(
+            y,
+            trend=self.trend,
+            damped=self.damped,
+            seasonal=self.seasonal,
+            seasonal_periods=self.seasonal_periods,
+        )
+
+        self._fitted_estimator = self._estimator.fit(
+            smoothing_level=self.smoothing_level,
+            optimized=self.optimized,
+            smoothing_slope=self.smoothing_slope,
+            smoothing_seasonal=self.smooting_seasonal,
+            damping_slope=self.damping_slope,
+            use_boxcox=self.use_boxcox,
+            remove_bias=self.remove_bias,
+            use_basinhopping=self.use_basinhopping,
+        )
+
+        self._is_fitted = True
+
+        return self
+
+    def predict(self, fh=None, X=None, return_conf_int=False, alpha=BaseForecasterOptionalFHinFit._DEFAULT_ALPHA):
```

Comment:
I'd prefer to import `_DEFAULT_ALPHA` at the top of the file from `sktime.forecasting.base`

Reply:
No problem. Would it be better to define defaults like this in a separate file rather than as a property of BaseForecaster? A default alpha could be applied in other contexts too.

Reply:
Sure, either way is fine for me.